tests/utils_tests/test_numberformat.py

```diff
@@ -57,6 +57,15 @@ def test_decimal_numbers(self):
         self.assertEqual(nformat(Decimal('1234'), '.', grouping=2, thousand_sep=',', force_grouping=True), '12,34')
         self.assertEqual(nformat(Decimal('-1234.33'), '.', decimal_pos=1), '-1234.3')
         self.assertEqual(nformat(Decimal('0.00000001'), '.', decimal_pos=8), '0.00000001')
+        self.assertEqual(nformat(Decimal('.000000000000000000999999'), '.', decimal_pos=2), '0.00')
```

Comment:
If you want to add additional tests that already work without this patch, it's okay, but please do it a separate commit so that it's clear which cases this fix is changing.